src/annotator/guest.js

```diff
@@ -282,6 +291,9 @@ export default class Guest {
     // Handlers for events sent when user hovers or clicks on an annotation card
     // in the sidebar.
     this.crossframe.on('focusAnnotations', (tags = []) => {
+      this._focusedAnnotations.clear();
```

Comment:
Possibly premature, but it might be useful/a safety precaution to encapsulate the reset-and-repopulate `_focusedAnnotations` logic if the two should "always go together."

Reply:
I'm not sure if that will always make sense for future use cases. For example if there was a signal coming in that meant "add X to the focused annotations" then we wouldn't want to clear first.